src/PointwiseFunctions/AnalyticSolutions/Poisson/Lorentzian.hpp

```diff
@@ -31,8 +31,11 @@ struct LorentzianVariables {
       ::Tags::Flux<Tags::Field<DataType>, tmpl::size_t<Dim>, Frame::Inertial>,
       ::Tags::FixedSource<Tags::Field<DataType>>>;
 
+  // NOLINTNEXTLINE(cppcoreguidelines-avoid-const-or-ref-data-members)
   const tnsr::I<DataVector, Dim>& x;
+  // NOLINTNEXTLINE(cppcoreguidelines-avoid-const-or-ref-data-members)
   const double constant;
+  // NOLINTNEXTLINE(cppcoreguidelines-avoid-const-or-ref-data-members)
   const double complex_phase;
```

Comment:
The clang-tidy warnings look right, at least for these last two.  Why should they be ignored?  Similar comments for the other file.

Reply:
I don't want functions in this class be able to modify this value. It's the user-defined value from options that gets passed down to this struct. I don't understand why clang-tidy want's me to remove the `const` when really I want it to be const.

Reply:
The [documentation](https://clang.llvm.org/extra/clang-tidy/checks/cppcoreguidelines/avoid-const-or-ref-data-members.html) links to the core guidelines rule and some extra reading.  I don't agree with everything in there, but that section seems pretty good.  The short answer for your class is that the functions already can't modify the value because they are const member functions.

Reply:
Ok makes sense, I removed the `const`